workshop/Lab0/README.md

```diff
@@ -0,0 +1,112 @@
+# Lab 0: Install Prequisites
+
+Before we begin, we need to install Service Catalog and a few other things
+we'll need. Towards that end, this lab contains instructions to install:
+
+* The Helm CLI, version 2.7.0 or later
```

Comment:
Nit, but for consistency, either make the first char of each line a cap or not.  So, if not, then lower case "The" on these lines, or uppercase "Create" at the end.